lib/components/regions_map/components/serach_places/interactor/view_models/search_places_bloc.dart

```diff
@@ -31,6 +34,18 @@ class SearchPlacesBloc extends Bloc<SearchPlacesEvent, SearchPlacesState> {
   Future<void> _onQueryTextChange(OnQueryTextChange event, Emitter<SearchPlacesState> emit) async {
     if (event.query.isNotEmpty) {
       emit(state.copyWith(pageState: PageState.success, showLinearIndicator: true));
+
+      final List<PredictionModel> regionMatches = [];
+      // If regions not empty search them by matches and aggregates
+      if (state.regions.isNotEmpty) {
+        for (final i in state.regions) {
+          if (i.title.contains(event.query)) {
+            regionMatches.add(PredictionModel(placeId: i.id, description: i.title));
```

Comment:
im not gonna block the PR but what reads better? 

regionMatches.add(PredictionModel(placeId: i.id, description: i.title));
or 
regionMatches.add(PredictionModel(placeId: region.id, description: region.title));

Reply:
Ok I'll fix in next pr